source/_docs/automation/trigger.markdown

```diff
@@ -378,7 +378,7 @@ automation:
       to: "error"
 ```
 
-Trigger on all state changes, but not attributes by setting `to` to `null`:
+Trigger on all state changes, but not attributes by setting `to` to `null`, `~`, or nothing:
```

Comment:
We don't use the tilde in our YAML documentation standards. This has been extensively discussed discussed and suggested in an architectural discussion (I prefer it, but is was voted against) and documented as such in our YAML style guide.

I suggest we revert this PR for that reason.

../Frenck

Reply:
OK, thank you for the catch. Let me fix it.